Assets/Scripts/Missions/MissionTutorial.cs

```diff
@@ -48,6 +48,9 @@ public class MissionTutorial : AMission
     [Header("Misc. Objects")]
     public List<MissionObject> extraObjects;
 
+    [Header("FMOD Audio")]
+    public FMODUnity.StudioEventEmitter MusicEv;
```

Comment:
Can we rename this to having a lowercase first letter? To follow naming conventions.

Reply:
I'd also just make this a private variable. We'd ideally only want to have variables we need to set in the inspector, as public!